docs/contents/proposals/secrets/secrets.md

```diff
@@ -1,22 +1,95 @@
+	EngineVersion string `json:"engineVersion,omitempty" aws:"EngineVersion"`
+	IOPS int64 `json:"iops,omitempty" aws:"Iops"`
+	KMSKeyID string `json:"kmsKeyID,omitempty" aws:"KmsKeyId"`
+	LicenseModel string `json:"licenseModel,omitempty" aws:"LicenseModel"`
+	MasterUserPassword string `json:"masterUserPassword,omitempty" aws:"MasterUserPassword"`
+	MasterUsername string `json:"masterUsername,omitempty" aws:"MasterUsername"`
+	MaxAllocatedStorage int64 `json:"maxAllocatedStorage,omitempty" aws:"MaxAllocatedStorage"`
+	MonitoringInterval int64 `json:"monitoringInterval,omitempty" aws:"MonitoringInterval"`
+	MonitoringRoleARN string `json:"monitoringRoleARN,omitempty" aws:"MonitoringRoleArn"`
+	MultiAZ bool `json:"multiAZ,omitempty" aws:"MultiAZ"`
+	OptionGroupName string `json:"optionGroupName,omitempty" aws:"OptionGroupName"`
+	PerformanceInsightsKMSKeyID string `json:"performanceInsightsKMSKeyID,omitempty" aws:"PerformanceInsightsKMSKeyId"`
+	PerformanceInsightsRetentionPeriod int64 `json:"performanceInsightsRetentionPeriod,omitempty" aws:"PerformanceInsightsRetentionPeriod"`
+	Port int64 `json:"port,omitempty" aws:"Port"`
+	PreferredBackupWindow string `json:"preferredBackupWindow,omitempty" aws:"PreferredBackupWindow"`
+	PreferredMaintenanceWindow string `json:"preferredMaintenanceWindow,omitempty" aws:"PreferredMaintenanceWindow"`
+	ProcessorFeatures []*ProcessorFeature `json:"processorFeatures,omitempty" aws:"ProcessorFeatures"`
+	PromotionTier int64 `json:"promotionTier,omitempty" aws:"PromotionTier"`
+	PubliclyAccessible bool `json:"publiclyAccessible,omitempty" aws:"PubliclyAccessible"`
+	StorageEncrypted bool `json:"storageEncrypted,omitempty" aws:"StorageEncrypted"`
+	StorageType string `json:"storageType,omitempty" aws:"StorageType"`
+	Tags []*Tag `json:"tags,omitempty" aws:"Tags"`
+	TDECredentialARN string `json:"tdeCredentialARN,omitempty" aws:"TdeCredentialArn"`
+	TDECredentialPassword string `json:"tdeCredentialPassword,omitempty" aws:"TdeCredentialPassword"`
+	Timezone string `json:"timezone,omitempty" aws:"Timezone"`
+	VPCSecurityGroupIDs []string `json:"vpcSecurityGroupIDs,omitempty" aws:"VpcSecurityGroupIds"`
+}
+```
+
+#### After ACK
+Now with Secrets integration in ACK, Alice creates a new [Kubernetes Secret]((https://kubernetes.io/docs/concepts/configuration/secret/)) and refers to the Secret when creating a manifest of the DB Instance. Alice's input will look as follows:
 
-Now with secrets integration in ACK, Alice creates a new Kubernetes secret and refers to the secret when creating a manifest of the DB Instance. When calling to describe the manifest, instead of her password printing explicitly in terminal, she sees the name of the secret reference that she created before. 
+```yaml
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBInstance
+metadata:
+  name: mydb
+spec:
+  dbInstanceClass: db.m1.large
+  dbInstanceIdentifier: mydb
+  engine: PostgreSQL
+  masterUserPasswordSecretRef:
```

Comment:
We can just call the field `masterUserPassword` instead of `masterUserPasswordSecretRef` (and yes, I know I had it as `masterUserPasswordSecretRef` in those slides :) )